README.md

```diff
@@ -27,7 +27,9 @@ In order to get started quickly with Fides, a sample project is bundled within t
 * [Docker](https://www.docker.com/products/docker-desktop) (version 20.10.11 or later)
 * [Python](https://www.python.org/downloads/) (version 3.8 through 3.10)
 
-#### Additional requirements (for ARM Mac Users)
+#### Additional requirements for optional dependencies (for ARM Mac Users)
```

Comment:
i considered just removing this section altogether, but then we'd have no trace about what you do need to do if you have this targeted use case. i could still easily be convinced to just remove this though, it feels a bit wordy and confusing as i have it!

Reply:
muchhhh better, thank you 🙏